### ISSUE TITLE
iPad compatibility

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/ios#2]
Add iPad compatibility, which would mean:
- Different screen layout to maximise map size
- Technical changes to switch this on in the code

Please add other changes as comments against this ticket

**Comment by mvl22** [comment]
@neilkachu Could you give an overview of what needs to be done to enable this? I'm aware that since the ticket was first added, the iOS platform has gone through a number of changes relating to dynamic layout.

**Comment by neilkachu** [comment]
Yes, all apps now should support 3.5, 4, 4.7, 5.4 & iPad sizes. All this is accomplished with the use of AutoLayout so that the UI's aren't just hardware scaled (Currently these devices will display a scaled version of the 4" size ). Currently all layout is frame based and this just can't do this. Every single UI layout has to be converted to Autolayout (in some cases this is easy) but is others it's not and realistically some UI's just don't work stretched out  (i.e. a full screen table view on an iPad ) So stage one would be to add AL support so CS is no longer scaled for iPhones, then for iPad we would need to look at the actual UI it will need to be different.
